app/assets/stylesheets/bootstrap-variables.scss

```diff
@@ -20,7 +20,7 @@ $brand-primary: darken(#0097FF,5%) !default; // darker creation-blue
 $brand-success: #8EDE3D !default;
 // $brand-info:            #5bc0de
 // $brand-warning:         #f0ad4e
-// $brand-danger:          #d9534f
+$brand-danger: #A80000;
```

Comment:
This was the original red color used both on mobile and on desktop. It was used on mobile next to BS3's default red. This is to unify the colors.